ani.ter-markosyan/repo/Lesson_3/Exercise_11/exercise_11.py

```diff
@@ -0,0 +1,3 @@
+width = float(input("Enter a widht of a field (in feet): "))
+lenght = float(input("Enter a lenght of a field (in feet): "))
+print("The area of a field is", round(width * lenght / 43560,2),"acres.")
```

Comment:
LW: What represents the number "43560". Use a constant variable to store the value.